desktop/renderer/index.js

```diff
@@ -75,8 +75,14 @@ class Keybase extends Component {
         return diffState
       }
 
-      sender.send('stateChange', getStore())
+      if (!sender.isDestroyed()) {
+        sender.send('stateChange', getStore())
+      }
       store.subscribe(() => {
+        if (sender.isDestroyed()) {
```

Comment:
:sheep: Not sure if it's a good idea to also unsubscribe from the store? store.subscribe returns a function you can call to unsubscribe the listener.